src/components/LogBookNewEvent.tsx

```diff
@@ -0,0 +1,85 @@
+import Button from 'react-bootstrap/Button';
+import { Form } from 'react-bootstrap';
+import { Formik, FormikHelpers } from 'formik';
+import { FC } from 'react';
+import LogBookNewEventProps from '../interfaces/LogBookEventProps';
+import LogBookFieldRow from './LogBookFieldRow';
+import EventRows from './EventRows';
+import LogBookHistoryRowI from '../interfaces/LogBookHistoryRowI';
+import EventRow from '../interfaces/EventRow';
+
+const LogBookNewEvent: FC<LogBookNewEventProps> = (props): JSX.Element => {
+  // Formik does not support useCallback
+  const handleFormSubmit = (
+    values: EventRow[],
+    actions: FormikHelpers<EventRow[]>
+  ): void => {
+    props.events.forEach((x) => values.push(x));
```

Comment:
x sijasta voisi käyttää kuvaavampaa nimeä, esim `event`. Sama homma oli myös EventRows.tsx.

Voit myös tuolla funktion parametreissa tiputtaa propsin pois jos avaat ne propsit näin `const LogBookNewEvent: FC<LogBookNewEventProps> = ({ events }): JSX.Element => {...}`